addons/stock/models/stock_config_setting_mixin.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class StockSettingMixin(models.AbstractModel):
+    _name = 'stock.config.setting.mixin'
+
+    module_delivery_dhl = fields.Boolean("DHL", help="Shipping connectors allow to compute accurate shipping costs, print shipping labels and request carrier picking at your warehouse to ship to the customer.")
```

Comment:
No need to add **help** in field definition. Please remove that wherever you added .